src/python/ddapp/ik.py

```diff
@@ -369,3 +369,61 @@ def tick(self):
 
         if self.handleAsyncTasks() > 0:
             return
+            
+    def runMultiRRT(self, grHand, qStart, xGoal, objectGrasped):
+        
+        commands = ['info = test_multiRRT_reach(r, s, \'%s\', %s, %s, %d);\n'%(grHand, qStart, xGoal, objectGrasped)]
+        self.comm.sendCommands(commands)
+
+        info = self.comm.getFloatArray('info')[0]
+        
+        return info
+        
+    def addAffordanceToHand(self, handLink, affordance, q, name):
```

Comment:
the arg handLink is actually a side string like "left" or "right"?  At least that appears to be what the matlab function is using, and the actually hand link names are hardcoded.